packages/ods/src/components/link/src/components/ods-link/ods-link.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../../../../style/link';
+
+.ods-link {
+  &__link {
+    @include ods-link;
+
+    &__text {
+      display: inline-flex;
+      align-items: center;
+    }
+
+    &__icon {
+      margin-left: var(--ods-spacing-8);
+      background-color: currentcolor;
+    }
+
+    &--disabled {
+      opacity: .5;
+      cursor: not-allowed;
+      color: var(--ods-color-neutral-400) !important;
```

Comment:
to avoid this, you can put the primary for not disabled case only, like:
```
&__link {
  $current: &;
  ...
  &:not(&#{$current}--disabled) {
      &#{$current}--primary {
        color: var(--ods-color-primary-500);

        &:hover {
          color: var(--ods-color-primary-700);
        }

        &:visited {
          color: var(--ods-color-primary-800);
        }
      }
    }
```